charts/internal/cilium/charts/hubble-relay/templates/cronjob.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: hubble-generate-certs
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble-generate-certs
```

Comment:
No hard opinion as not all components have it, but you can also add the label `gardener.cloud/role: system-component`.